package.json

```diff
@@ -27,6 +27,10 @@
     "prettier": "^1.18.2",
     "webpack": "^4.43.0"
   },
+  "peerDependencies": {
+    "preact": "^10.4.0",
```

Comment:
Since we use the `options.hook` second and thirth argument we have advanced to requiring 10.4.0 when using hooks, we could probably detect this not being there